definitions/consented/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -985,5 +985,152 @@
     "CaseEventID": "FR_updateCourtInfo",
     "UserRole": "caseworker-divorce-financialremedy-judiciary",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
```

Comment:
just to make sure, can all of these go to prod?

Reply:
these will yes, have been signed off, but the pipeline is having issues

Reply:
yep they are all signed off to go in